webapp/tokenList/stakeTokens.ts

```diff
@@ -25,7 +27,7 @@ const websitesMap: Partial<Record<StakeExtensions['protocol'], string>> = {
 // TODO: Some tokens are not deployed, so their rewards can't be configured
 // https://github.com/hemilabs/ui-monorepo/issues/752#issuecomment-2616916547
 export const stakeWhiteList: Partial<
-  Record<Token['chainId'], Record<Address, StakeExtensions>>
+  Record<Token['chainId'], Record<Address | string, StakeExtensions>>
```

Comment:
This is the same as just doing

```suggestion
  Record<Token['chainId'], Record<string, StakeExtensions>>
```

Reply:
Indeed